### PR TITLE
Cleanup var naming in ISOProperties

### PR DESCRIPTION
**Pull request by Helios747**
Was poking around in ISOProperties and noticed that a lot of the vars weren't up to date with naming guidelines.

### CODE REVIEW

File: Source/Core/DolphinWX/ISOProperties/ISOProperties.h [hunk 1]
```diff
@@ -67,26 +67,26 @@ class CISOProperties : public wxDialog
   PHackData m_PHack_Data;
```
Comment (lioncash) [inline Source/Core/DolphinWX/ISOProperties/ISOProperties.h]: were these two variables missed?

Comment (Helios747) [reply Source/Core/DolphinWX/ISOProperties/ISOProperties.h]: oops. gottem

**Comment by shuffle2** [comment]
honestly it would be great if this could be done across the entire codebase in an automated way(?)

**Comment by Helios747** [comment]
Tricky, because that assumes that all non-conforming vars are named the same way or similar ways.

For example, the DCBZOFF and FPRF vars are just acronyms all capped. I imagine there are some typos in var names too.

I think we could probably hit a lot of them just by somehow detecting CamelCase in class vars though.

Worth noting that I did re-word one or two vars just because they had dumb names that would have translated to absurdly long var names.

**Pull request merged**